### PR TITLE
fix: grammatical issues in testing_strategies

### PR DESCRIPTION
**Pull request by yashikabhargava**
### Description

Fixed minor grammatical issues in testing_strategies.

### CODE REVIEW

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md [hunk 1]
```diff
@@ -50,22 +50,22 @@ In the following sections, we'll build up a support chart in this format.
 
 This typically involves making educated guesses based on user demographics. For example, suppose your users are in North America and Western Europe:
 
-A quick online search tells you that most people in North America and Western Europe use Windows or Mac desktops/laptops, where the main browsers are Chrome, Firefox, Safari, and Edge. You'd probably want to just test the latest versions of these browsers, as these browsers get regular updates. These should all go in the A grade tier.
+A quick online search tells you that most people in North America and Western Europe use Windows or Mac desktops/laptops, where the main browsers are Chrome, Firefox, Safari, and Edge. You'd probably want to just test the latest versions of these browsers, as these browsers get regular updates. These should all go in the A-grade tier.
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:53]: There many other use of A grade, … in this section. This should be unified.

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md [hunk 2]
```diff
@@ -50,22 +50,22 @@ In the following sections, we'll build up a support chart in this format.
...
+For IE, you'd probably want to test the last couple of versions. Some people still use IE 9. Since IE is old and less capable, and Microsoft is phasing out IE support, let's put it in the B-grade tier.
 
 Opera Mini [isn't very capable of running complex JavaScript](https://dev.opera.com/articles/opera-mini-and-javascript/), so we should put this into grade B as well.
 
 Thus, we've based our choice of which browsers to test on the browsers that we expect our users to use.
 This gives us the following support chart so far:
 
-1. A grade: Chrome and Firefox for Windows/Mac, Safari for Mac, Edge for Windows, iOS Safari for iPhone/iPad, Android stock browser (last two versions) on phone/tablet, Chrome and Firefox for Android (last two versions) on phone/tablet
+1. A grade: Chrome and Firefox for Windows/Mac, Safari for Mac, Edge for Windows, iOS Safari for iPhone/iPad, Android stock browser (last two versions) on phone/tablet, Chrome, and Firefox for Android (last two versions) on phone/tablet
 2. B grade: IE for Windows, Opera Mini
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:65]: Remove IE.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md (hunk 1 above)
Comment (yashikabhargava) [reply files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:53]: Yes, I missed some of those. Fixing it.

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md (hunk 2 above)
Comment (yashikabhargava) [reply files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:65]: I'm sorry, I did not get it. Do I need to remove IE from line 65?

Comment (teoli2003) [reply files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md]: Yes, please. IE is dead; we should not recommend testing for it anymore.

Comment (yashikabhargava) [reply files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md]: Okay, got it.

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md [hunk 3]
```diff
@@ -50,22 +50,22 @@ In the following sections, we'll build up a support chart in this format.
...
+A quick online search tells you that most people in North America and Western Europe use Windows or Mac desktops/laptops, where the main browsers are Chrome, Firefox, Safari, and Edge. You'd probably want to just test the latest versions of these browsers, as these browsers get regular updates. These should all go in the A-grade tier.
 
 Most people in this demographic also use either iOS or Android phones, so you'd probably want to test the latest versions of iOS Safari, the last couple of versions of the old Android stock browser, and Chrome and Firefox for iOS and Android. You should ideally test these on both a phone and a tablet, to ensure responsive designs work.
 
-For IE, you'd probably want to test the last couple of versions. Some people still use IE 9. Since IE is old and less capable, and Microsoft is phasing out IE support, let's put it in the B grade tier.
+For IE, you'd probably want to test the last couple of versions. Some people still use IE 9. Since IE is old and less capable, and Microsoft is phasing out IE support, let's put it in the B-grade tier.
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:58]:
```suggestion
```

File: files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md [hunk 4]
```diff
@@ -115,13 +115,13 @@ Also, you should be aware of situation-specific needs. For example, if you are c
 
 So, our final support chart will end up looking like so:
 
-1. A grade: Chrome and Firefox for Windows/Mac, Safari for Mac, Edge and IE for Windows (last two versions of each), iOS Safari for iPhone/iPad, Android stock browser (last two versions) on phone/tablet, Chrome and Firefox for Android (last two versions) on phone tablet. Accessibility passing common tests.
-2. B grade: IE 8 and 9 for Windows, Opera Mini.
-3. C grade: Opera, other niche modern browsers.
+1. A-grade: Chrome and Firefox for Windows/Mac, Safari for Mac, Edge, and IE for Windows (last two versions of each), iOS Safari for iPhone/iPad, Android stock browser (last two versions) on phone/tablet, Chrome, and Firefox for Android (last two versions) on phone tablet. Accessibility passing common tests.
+2. B-grade: IE 8 and 9 for Windows, Opera Mini.
```
Comment (teoli2003) [inline files/en-us/learn/tools_and_testing/cross_browser_testing/testing_strategies/index.md:119]:
```suggestion
1. A-grade: Chrome and Firefox for Windows/Mac, Safari for Mac, and Edge (last two versions of each), iOS Safari for iPhone/iPad, Android stock browser (last two versions) on phone/tablet, Chrome, and Firefox for Android (last two versions) on phone tablet. Accessibility passing common tests.
2. B-grade: Opera Mini.
```

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**